doc/design/TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+---
+title: template
+authors:
+  - template
+reviewers:
+  - template
+creation-date: DD-MM-YYYY
+last-updated: DD-MM-YYYY
+---
+
+# Title
+
+## Summary
+
+## Motivation
+
+### Use Cases
+
+### Goals
+
+### Non-Goals
+
+## Proposal
+
+### Workflow Description
+
+### API Extensions
+
+### Implementation Details/Notes/Constraints
+
+### Test Plan
```

Comment:
lets have : Update  & Backward Compatibility considerations

Reply:
I like it :)